library/src/math/cancel.rs

```diff
@@ -0,0 +1,171 @@
+use super::*;
+
+/// Displays a diagonal line over math content.
+///
+/// ## Example
+///
+/// ```example
+/// Here, we can simplify: $ (a dot.c b dot.c cancel(x)) / cancel(x) $
+///
+/// We can also invert the line: $ (a cancel((b + c), invert: #true)) / cancel(b + c, invert: #true) $
+///
+/// Or draw two lines (a cross): $ x + cancel(a + b + c, cross: #true) $
+///
+/// Or set its length, relative to the diagonal: $cancel(x, length: #200%)$
+///
+/// Rotating and even making it red and thicker is possible: $ cancel(x, rotate: #30deg, stroke: #{red + 1.5pt}) $
+/// ```
```

Comment:
Text describing the examples should mostly not be part of the example itself. It would also be great if the example would occupy less columns. If you look at some of the existing docs, you can see that they rarely exceed 40 columns or so. That way, they fit side-to-side in the rendered output. It would still be great if you could reduce this example to its essence. In particular, the examples that apply to a certain argument should live with the argument (they then get the example dropdown).

I know it would be much easier if you could build the docs yourself and we want to make that possible in the medium term. I also make a pass over the docs changes myself before each release, but it's great if its already mostly in shape by then. :)

Reply:
Gotcha, thanks! I've improved this now.